web/server/console.py

```diff
@@ -100,3 +100,9 @@ def log_test_results(
     def log_success(self, msg: str) -> None:
         self.queue.put_nowait(self._make_event(msg))
         self.stop_snapshot_progress()
+
+    def log_exception(self, tb: str) -> None:
+        self.queue.put_nowait(self._make_event(tb, event="errors", ok=False))
```

Comment:
can we have traceback as property in payload in case we need to add more details to describe error:
```
{
    "message": "Plan Failed",
    "traceback": "..."
}
```